### PR TITLE
Bug 1209718 - New Utility Tray is not screen reader friendly.

### PR DESCRIPTION
**Pull request by mcav**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cc837e924e8acdcd8dad77acee38c08548635f15

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c51305631b8f173c6c571b65c5194a1cbd844438

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=80e2fe4dc59374c89d36092c0d09b8ac57980f0a

File: build/csslint/xfail.list [hunk 1]
```diff
@@ -93,7 +93,7 @@ apps/system/style/themes/default/buttons.css 3 0
 apps/system/style/themes/default/core.css 4 5
 apps/system/style/ttlview/ttlview.css 1 0
 apps/system/style/update_manager/update_manager.css 4 0
-apps/system/style/utility_tray/utility_tray.css 3 0
+apps/system/style/utility_tray/utility_tray.css 4 0
```
Comment (etiennesegonzac) [inline build/csslint/xfail.list]: out of curiosity, what is bugging the linter? the var itself? changing it's value?

Comment (mcav) [inline build/csslint/xfail.list]: It thinks rules with only CSS variable declarations are "empty":
```
.foo {
 --my-var: whatever;
}
```

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=29c2dab140aae5099b8ba4d66b16e7d754b93a2c

File: build/csslint/xfail.list (hunk 1 above)
Comment (etiennesegonzac) [inline build/csslint/xfail.list]: ahah, ok, good to know :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7b6ce825b3816d8717780651a387959f04367bd5

**Pull request merged**